docs/configurations.md

```diff
@@ -101,47 +127,52 @@ The following describes the configurations available for each storage.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+</div>
+</div>
 
 ### ScalarDB Server (gRPC)
 
-[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB.
-To interact with ScalarDB Server, you must specify `grpc` for the `scalar.db.transaction_manager` property.
+[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB. To interact with ScalarDB Server, you must specify `grpc` as the value for the `scalar.db.transaction_manager` property.
 
-The following configurations are available for ScalarDB Server:
+The following configurations are available for ScalarDB Server using gRPC transactions:
 
-| Name                                       | Description                                                 | Default                |
-|--------------------------------------------|-------------------------------------------------------------|------------------------|
-| `scalar.db.transaction_manager`            | `grpc` should be specified.                                 | -                      |
-| `scalar.db.contact_points`                 | ScalarDB Server host.                                       |                        |
-| `scalar.db.contact_port`                   | Port number for ScalarDB Server.                            | `60051`                |
-| `scalar.db.grpc.deadline_duration_millis`  | The deadline duration for gRPC connections in milliseconds. | `60000` (60 seconds)   |
-| `scalar.db.grpc.max_inbound_message_size`  | The maximum message size allowed for a single gRPC frame.   | The gRPC default value |
-| `scalar.db.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.        | The gRPC default value |
+| Name                                       | Description                                                 | Default                 |
+|--------------------------------------------|-------------------------------------------------------------|-------------------------|
+| `scalar.db.transaction_manager`            | `grpc` should be specified.                                 | -                       |
+| `scalar.db.contact_points`                 | ScalarDB Server host.                                       |                         |
+| `scalar.db.contact_port`                   | Port number for ScalarDB Server.                            | `60051`                 |
+| `scalar.db.grpc.deadline_duration_millis`  | The deadline duration for gRPC connections in milliseconds. | `60000` (60 seconds)    |
+| `scalar.db.grpc.max_inbound_message_size`  | The maximum message size allowed for a single gRPC frame.   | The gRPC default value. |
+| `scalar.db.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.        | The gRPC default value. |
 
 For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
```

Comment:
I don't think this section should be in the docs in the `master`, `3.10`, and `3.9` branches since ScalarDB Server has been deprecated as of ScalarDB 3.9.

For ScalarDB docs on the `master`, `3.10`, and `3.9` branches, should we remove this section or point users to [ScalarDB Cluster Configurations](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/scalardb-cluster-configurations.md)?

Reply:
> I don't think this section should be in the docs in the 3.10 and 3.9 branches since ScalarDB Server has been deprecated as of ScalarDB 3.9.

Yes, please remove this. We have other documents that mentions ScalarDB Server as well. I think we should also remove them.

Reply:
Got it. I will make those changes in this doc and related docs on branches `master`, `3.10`, and `3.9` after this PR is merged.